### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by fcavalieri**

### CODE REVIEW

**Comment by fcavalieri** [comment]
Hello @dknochen , could you review this with high priority? Thanks

File: samples/GetFacts/GetFacts/Program.cs [hunk 1]
```diff
@@ -7,34 +7,37 @@
...
-            foreach (dynamic fact in factTable.Children())
-            {
-                Console.WriteLine("===========================");
-                Console.WriteLine("Fact " + i + ":");
-                Console.WriteLine(JsonConvert.SerializeObject(fact, Formatting.Indented));
-                i++;
-            }
+      // list some facts
+      dynamic factsResonse = dataAPI.ListFacts(token, ticker: new List<string> { "t" }, 
+                                                      fiscalYear: new List<string> { "2014" },
```
Comment (dknochen) [inline samples/GetFacts/GetFacts/Program.cs]: could we add another fiscalYear to see that it works for multiple values?

Comment (fcavalieri) [inline samples/GetFacts/GetFacts/Program.cs]: done

**Comment by fcavalieri** [comment]
Back to you @dknochen :)

File: build.Test/DataApiTests.cs [hunk 2]
```diff
@@ -0,0 +1,850 @@
...
+using System.Linq;
+using System.Reflection;
+using RestSharp;
+using NUnit.Framework;
+
+using CellStore.Client;
+using CellStore.Api;
+using CellStore.Model;
+
+namespace CellStore.Test
```
Comment (dknochen) [inline build.Test/DataApiTests.cs:14]: are these tests also run?

**Comment by dknochen** [comment]
looks very good :)
feel free to merge

**Comment by dknochen** [comment]
Why is release 2.0.0 in branch release-1.1.0?

**Comment by fcavalieri** [comment]
Originally this PR was meant to be just regenerating the bindings with the
latest documentation. But then I was forced to introduce the breaking
changes for the parameters cardinality because nothing was working anymore.

File: build.Test/DataApiTests.cs (hunk 2 above)
Comment (fcavalieri) [inline build.Test/DataApiTests.cs:14]: not yet, i am not even sure how we should write the tests

**Pull request merged**